### PR TITLE
fix typos in the derived fields docs

### PR DESCRIPTION
**Pull request by Ahimasjason**
Changes Made:
- Renamed the derived name from "date_rfc3339" to "duration_rfc3339" to be consistent with the schema.

```graphql
type Query {
	duration_rfc2822(arg: String): DateRFC2822!
	duration_rfc3339(arg: String): DateRFC3339!
}
```

### CODE REVIEW

File: docs/en/src/derived_fields.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ struct Query;
 
 #[Object]
 impl Query {
-    #[graphql(derived(name = "date_rfc3339", into = "DateRFC3339"))]
+    #[graphql(derived(name = "duration_rfc3339", into = "DateRFC3339"))]
     async fn duration_rfc2822(&self, arg: String) -> DateRFC2822 {
```
Comment (sunli829) [inline docs/en/src/derived_fields.md:44]: I think `Query::duration_rfc2822` should be renamed to `Query::date_rfc2822`

Comment (Ahimasjason) [reply docs/en/src/derived_fields.md:44]: Ok, will fix it.

Comment (Ahimasjason) [reply docs/en/src/derived_fields.md]: @sunli829  i have renamed `Query::duration_rfc2822` to `Query::date_rfc2822`

Comment (sunli829) [reply docs/en/src/derived_fields.md]: The field `duration_rfc2822` should also be renamed to `date_rfc2822` 🙂

Comment (Ahimasjason) [reply docs/en/src/derived_fields.md]: I got it! Thank you @sunli829 for clarifying , I have fixed those changes.

**Comment by sunli829** [comment]
thanks! 🙂

**Pull request merged**